application/F3DOptionsTools.cxx

```diff
@@ -225,9 +225,10 @@ void PrintHelp(const std::string& execName, const cxxopts::Options& cxxOptions)
   }
 
   f3d::log::info("\nExamples:");
-  for (const auto& [cmd, desc] : examples)
+  for (const auto& example : examples)
   {
-    ::PrintHelpPair(cmd, desc, 50);
```

Comment:
this should still work afaics

Reply:
yes, you can just use this method instead of printing manually

Reply:
I tried it but it gets broken again. I am attaching a screenshot.
![Screenshot 2024-09-16 093151](https://github.com/user-attachments/assets/e191a3b4-d381-419a-b355-bcfdbd900b2c)

Reply:
you could add a `separator` arg to PrintHelpPair at the end that default to `''`  and set it to `\n` in your case ?

Reply:
That seems to do the trick! Thanks!